sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,17 +1,21 @@
 # Release History
 
-## 5.1.0-beta.6 (Unreleased)
+## 5.1.0-beta.6 (2021-05-18)
 
-New Features:
-    - We are now targeting the service's v3.1-preview.5 API as the default instead of v3.1-preview.4.
-    - `beginAnalyzeActions` adds support for analyze sentiment actions.
-    - `disableServiceLogs` parameter is added to most actions in `beginAnalyzeActions` that controls logging.
+### New Features
 
-Breaking Changes:
-    - `beginAnalyzeBatchActions` was renamed to `beginAnalyzeActions`.
+- We are now targeting the service's v3.1-preview.5 API as the default instead of v3.1-preview.4.
+- `beginAnalyzeActions` adds support for analyze sentiment actions.
+- `disableServiceLogs` parameter is added to most actions in `beginAnalyzeActions` that controls logging.
```

Comment:
should you say "service-side logging" here to not confuse with client logging?

Reply:
good point! edited.